packages/opentelemetry-exporter-collector/src/CollectorExporterBase.ts

```diff
@@ -98,16 +100,29 @@ export abstract class CollectorExporterBase<
   /**
    * Shutdown the exporter.
    */
-  shutdown(): void {
+  shutdown(): Promise<void> {
```

Comment:
Use async/await? We should try to be consistent on if we use async/await, promise.then, or callbacks. Seems like we are trending towards async/await so we should try to keep that.

Reply:
Please analyze the function itself what it does, and especially if you want to handle calling shutdown multiple times and still have correct behaviour to be notified once

Reply:
That behavior could still be achieved with async/await, but it's fine if you prefer this style.